src/Xfx.Controls.Droid/Renderers/XfxEntryRendererDroid.cs

```diff
@@ -190,18 +190,52 @@ private void SetUnderlineColor(AColor color)
 
         private void SetHintLabelActiveColor(AColor color)
         {
-            var hintText = Control.Class.GetDeclaredField("mFocusedTextColor");
-            hintText.Accessible = true;
-            hintText.Set(Control, new ColorStateList(new int[][] { new[] { 0 } }, new int[] { color }));
+            // design library 28.0.0 and later changed the field from mFocusedTextColor to focusedTextColor.
+            // Assume using design library >= 28 by checking for the new field name first.
+            var field = GetDeclaredField("focusedTextColor") ?? GetDeclaredField("mFocusedTextColor");
```

Comment:
Would it be better to use :
```c#
Java.Lang.Reflect.Field field = null;
if (Android.OS.Build.VERSION.SdkInt == Android.OS.BuildVersionCodes.P)
{
    field = GetDeclaredField("focusedTextColor");
}
else
{
    field = GetDeclaredField("mFocusedTextColor");
}
```

Reply:
I agree, please ensure you're checking the sdk version and proceed accordingly

Reply:
The SDK version gives you the version of the OS that the app is running on and not the version of the design / support libraries that the app was built with.  So for example, if I build with the 28.0 design / support libraries (which has the field changes) but run the app on my hardware which is Android 6 Marshmallow then the issue is still present as `Android.OS.Build.VERSION.SdkInt` will return `Android.OS.BuildVersionCodes.M`.

Reply:
Ah, I see! So while `Android.OS.Build.VERSION.SdkInt` returns the build version of the **device** the app is running on, there's no actual way of determining the build tools version was used to build the app? I suppose if in version 28.0.0 onwards the field name was changed, then there isn't really another way of getting the field `(mF/f)ocusedTextColor` except checking if `GetDeclaredField()` returns `null`...

Reply:
Correct.